File: Assignment_39/program1.cpp
//////////////////////////////////////////////////////////////////////////
//
// Name :          program1.c
// Discription :   Get input and Display 
// Output :        Integer 
// Author :        Shivam Rajendra Kale
// Date :          22/06/2025
//
//////////////////////////////////////////////////////////////////////////

/* 
  Start
    Accept input from user
      Display input in generic     
  Stop    
*/

#include<iostream>
using namespace std;

template <class T>
T Display(T Value, int iSize)
{
    int i = 0;

    for(i = 1; i <= iSize; i++)
    {
        cout<<Value<<"\t";
    }
    cout<<"\n";
}

int main()
{
    Display('M', 7);

    Display(10.45f, 5);
    
    return 0;
}

File: Assignment_52/program1.java
//////////////////////////////////////////////////////////////////////////
//
// Name :          program1.java
// Discription :   use to Display elements of Array
// Input :         Integer
// Output :        Integer
// Author :        Shivam Rajendra Kale
// Date :          16/10/2025
//
//////////////////////////////////////////////////////////////////////////

import java.util.*;

class MyArray
{
    public void Display(int Arr[], int Brr[])
    {
        int i = 0;

        System.out.println("Elements of first Array are : ");
        for(i = 0; i < Arr.length; i++)
        {
            System.out.print(Arr[i]+"\t");
        }

        System.out.println("\nElements of second Array are : ");
        for(i = 0; i < Brr.length; i++)
        {
            System.out.print(Brr[i]+"\t");
        } 
    }
}

class program1
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter isze of first Array : ");
        int iSize = sobj.nextInt();

        int Arr[] = new int [iSize];

        System.out.println("Enter elements : ");
        for(int i = 0; i < iSize; i++)
        {
            Arr[i] = sobj.nextInt();
        }

        System.out.println("Enter isze of second Array : ");
        int iSize1 = sobj.nextInt();

        int Brr[] = new int [iSize1];

        System.out.println("Enter elements : ");
        for(int i = 0; i < iSize1; i++)
        {
            Brr[i] = sobj.nextInt();
        }

        MyArray mobj = new MyArray();

        mobj.Display(Arr, Brr);

        sobj.close();
    }
}